runbotics/runbotics-orchestrator-ui/src/src-app/layouts/MainLayout/TopBar/index.tsx

```diff
@@ -76,10 +87,13 @@ const TopBar: FC<TopBarProps> = ({ className, ...rest }) => {
     const isTenantAdmin = useRole([Role.ROLE_TENANT_ADMIN]);
     const isAdmin = useRole([Role.ROLE_ADMIN]);
     const isGuest = user?.roles.includes(Role.ROLE_GUEST);
+    const isJustUser = user?.roles.every((role) => role === Role.ROLE_USER);
```

Comment:
couldn't we create a helper hook that will return all of these? Or extend useAuth to actually return: `isTenantAdmin`, `isAdmin`, `isGuest`, `isJustUser` ?

Reply:
btw. I don't like `isJustUser`. Go for `isUser` and `isRpaUser` if needed somewhere else